data/movie/Left Behind (2014)/2015-eone-canada-blu-ray/disc01.json

```diff
@@ -0,0 +1,497 @@
+{
+  "Index": 0,
+  "Name": "Left Behind",
```

Comment:
What version of ImportBuddy did you use to generate this file? (assuming you used ImportBuddy). Just wondering since there are propertys like Slug and Disc ContentHash that are missing

Reply:
This was with 1.1 that I downloaded today. It did display a message about calculating the content hash so it's weird it didn't copy it.

Reply:
Ah, I think I know the issue - I just saw this behavior recently. Is the drive letter correct in the appsettings.json? I think if this doesn't match the letter of the drive on your system it will not be able to calculate the disc hash

Reply:
The drive letter is correct. Is it possible the drive index being incorrect could cause this?
```json
"Drives": [
   {
      "Index": 0,
      "Letter": "E",
      "Name": "Optical Drive"
   }
]
```

Reply:
Probably not the drive index - that is only  used by MakeMkv and not calculating the disc hash. No need to block this PR on this - I'll open a bug and investigate why this happens

Reply:
I just retried to see if I could get it to make a has and got an error. I'll add it to an issue.